src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -46,110 +59,80 @@ const propTypes = {
         source: PropTypes.string,
     }),
 
-    /** Does this fragment belong to a reportAction that has not yet loaded? */
-    loading: PropTypes.bool,
-
-    /** The reportAction's source */
-    source: PropTypes.oneOf(['Chronos', 'email', 'ios', 'android', 'web', 'email', '']),
+    /** Should the comment have the appearance of being grouped with the previous comment? */
+    displayAsGroup: PropTypes.bool.isRequired,
 
-    /** Should this fragment be contained in a single line? */
-    isSingleLine: PropTypes.bool,
+    // Properties required for a TEXT fragment
 
-    // Additional styles to add after local styles
-    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+    /** User's accountID */
+    accountID: PropTypes.number.isRequired,
 
     /** The accountID of the copilot who took this action on behalf of the user */
     delegateAccountID: PropTypes.number,
 
-    /** icon */
-    actorIcon: avatarPropTypes,
-
-    ...windowDimensionsPropTypes,
+    /** Should this fragment be contained in a single line? */
+    isSingleLine: PropTypes.bool,
 
-    /** localization props */
-    ...withLocalizePropTypes,
+    /** The actor's icon */
+    actorIcon: avatarPropTypes,
 };
 
 const defaultProps = {
-    isAttachment: false,
-    attachmentInfo: {
+    source: '',
+    pendingAction: undefined,
+    isUploadedAttachment: false,
+    uploadedAttachmentInfo: {
         name: '',
         size: 0,
         type: '',
         source: '',
     },
-    loading: false,
-    isSingleLine: false,
-    source: '',
-    style: [],
     delegateAccountID: 0,
+    isSingleLine: false,
     actorIcon: {},
 };
 
 function ReportActionItemFragment(props) {
-    switch (props.fragment.type) {
-        case 'COMMENT': {
-            // If this is an attachment placeholder, return the placeholder component
-            if (props.isAttachment && props.loading) {
```

Comment:
This is a less typical point where an attachment could be rendered, but actually **I wasn't able to trigger this path**